src/templates/next/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,17 @@
+import { NotificationProps } from "~/common/Notification"
+import BaseParagraph from "../shared/Paragraph"
+import { RxCross2 } from "react-icons/rx"
+
+const Notification = ({ title }: NotificationProps) => {
+  return (
+    <div className="flex flex-col sm:flex-row justify-between gap-6 bg-[#FEF4E8] py-4 px-5 sm:px-10">
+      <BaseParagraph className="text-base " content={title}></BaseParagraph>
+      <button className="flex flex-row ml-auto items-center gap-1">
```

Comment:
suggestion: the button should ideally work with the `useState` hook in React to hide the entire notification